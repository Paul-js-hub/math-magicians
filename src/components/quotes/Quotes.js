import React from 'react';
import quotes from './quotes.module.css';

const Quotes = () => (
  <div className={quotes.quotes}>
    <p>
      Without mathematics, thereâ€™s nothing you can do. Everything around you is
      mathematics. Everything around you is numbers. -Shakuntala Devi
    </p>
  </div>
);

export default Quotes;
