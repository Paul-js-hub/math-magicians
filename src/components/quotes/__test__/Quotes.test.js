import { render, screen, fireEvent} from '@testing-library/react';
import renderer from 'react-test-renderer';
import Quotes from '../Quotes.js';

test('it returns a calculator component', () => {
  const tree = renderer.create(<Quotes />).toJSON();
  expect(tree).toMatchSnapshot();
});