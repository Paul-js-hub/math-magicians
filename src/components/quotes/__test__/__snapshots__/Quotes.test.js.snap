// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it returns a calculator component 1`] = `
<div
  className="quotes"
>
  <p>
    Without mathematics, thereâ€™s nothing you can do. Everything around you is mathematics. Everything around you is numbers. -Shakuntala Devi
  </p>
</div>
`;
